test/expected/ddl_single.out

```diff
@@ -339,14 +329,27 @@ FOR EACH STATEMENT EXECUTE PROCEDURE empty_trigger_func();
 --expect error cases
 \set ON_ERROR_STOP 0
```

Comment:
Get rid of ON_ERROR_STOP 0

Reply:
Shouldn't rename column generate error? Can remove, but doesn't seem related to my changes.